util/schema/registry_schema.json

```diff
@@ -415,7 +415,37 @@
 			"suggest": false
 		},
 		"usages": {
-			"suggest": false
+			"suggest": false,
+      			"level": "error",
+      			"description": "Information on usage of particular ontology.",
+			"type": "array",
+			"items": {
+        			"type": "object",
+        			"properties": {
+					"user": {
+          			      		"type": "string",
+          					"format": "uri"
+       				 	},
+					"type": { "type": "string" },
+					"description": {"type":"string"},
+        				"example": {
```

Comment:
Since this is an array, it should be named "examples".

Reply:
From the issue in #1008, it was saying to use "example".

Reply:
Since there are good reasons to allow more than one example, let's go with "examples".